AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -1932,9 +1932,9 @@ hnonline.sk,digitaltrends.com#%#//scriptlet("abort-current-inline-script", "docu
 !+ PLATFORM(windows, mac, android)
 hnonline.sk,digitaltrends.com#%#AG_abortInlineScript(/googlefc/, 'document.addEventListener');
 !+ NOT_PLATFORM(windows, mac, android)
-news247.gr,sport24.gr,contra.gr,esquire.com.gr,slickdeals.net,narcity.com,observador.pt,ni.dk,jetphotos.com,marketscreener.com,zonebourse.com,ign.com,dziennikbaltycki.pl,marca.com,snopes.com,cnnturk.com,polskatimes.pl,abc.es,dumpert.nl,accuweather.com,expressilustrowany.pl#%#//scriptlet("abort-current-inline-script", "atob", "googlefc")
+klix.ba,sport24.gr,contra.gr,esquire.com.gr,slickdeals.net,narcity.com,observador.pt,ni.dk,jetphotos.com,marketscreener.com,zonebourse.com,ign.com,dziennikbaltycki.pl,marca.com,snopes.com,cnnturk.com,polskatimes.pl,abc.es,dumpert.nl,accuweather.com,expressilustrowany.pl#%#//scriptlet("abort-current-inline-script", "atob", "googlefc")
 !+ PLATFORM(windows, mac, android)
-news247.gr,sport24.gr,contra.gr,esquire.com.gr,slickdeals.net,narcity.com,observador.pt,ni.dk,jetphotos.com,marketscreener.com,zonebourse.com,ign.com,dziennikbaltycki.pl,marca.com,snopes.com,cnnturk.com,polskatimes.pl,abc.es,dumpert.nl,accuweather.com,expressilustrowany.pl#%#AG_abortInlineScript(/googlefc/, 'atob');
+klix.ba,news247.gr,sport24.gr,contra.gr,esquire.com.gr,slickdeals.net,narcity.com,observador.pt,ni.dk,jetphotos.com,marketscreener.com,zonebourse.com,ign.com,dziennikbaltycki.pl,marca.com,snopes.com,cnnturk.com,polskatimes.pl,abc.es,dumpert.nl,accuweather.com,expressilustrowany.pl#%#AG_abortInlineScript(/googlefc/, 'atob');
```

Comment:
redundant JS rule and unnecessary hints.
Better copy `! 'contributor.google.com'` from Base filter.